jwst/pixel_replace/pixel_replace.py

```diff
@@ -384,7 +392,7 @@ def mingrad(self, model):
         newdq = model_replaced.dq
 
         # Make an array of x/y values on the detector
-        xsize, ysize = self.input.meta.subarray.xsize, self.input.meta.subarray.ysize
+        ysize, xsize = indata.shape
```

Comment:
Based on lessons learned from https://github.com/spacetelescope/jwst/pull/7869, I think this will fail on TSO data as-is.  (Not that TSO would likely want to run this routine)

Reply:
Yep, always safer to do `ysize, xsize = indata.shape[-2:]` just in case.

Reply:
This was a local change to test mingrad on the JIRA ticket data, and not intended to be part of this PR - I've undone the change.

Reply:
@tapastro - it looks like xsize and ysize were accidentally reversed, when reverting the change.